requirements.txt

```diff
@@ -55,3 +54,4 @@ bcrypt
 flask-bcrypt
 mock
 hdfs
+impyla
```

Comment:
that's where the merge conflict is coming from, I decided to remove the `requirements.txt` as it was duplicating information with the setup.py, and was all-or-nothing which won't work for anyone.

I hooked up this new utility which is pretty sweet:
https://requires.io/github/airbnb/airflow/requirements/?branch=pep8